### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by gargsanchit31**
1. added faqs.
2. updated intro for speaker diarize and interaction apis.
3. changed order of apis.
4. added period to sentences.

### CODE REVIEW

File: docs/interaction-analytics-api.md [hunk 1]
```diff
@@ -215,150 +221,150 @@ print(response.text)
...
+| Parameter     | Type   | Description                                            | Notes |
+| ------------- | ------ | ------------------------------------------------------ | ----- |
+| speaker_id    | String | The speaker id for the corresponding audio segment.    |       |
+| start         | Number | Start time of the audio segment in seconds.            |       |
+| end           | Number | End time of the audio segment in seconds.              |       |
+| text          | String | The transcription output corresponding to the segment. |       |
+| confidence    | Number | The confidence score for the transcribed segment.      |       |
+| emotion       | String | Emotion corresponding to the segment.                  | Will be computed only if specified via metrics. |
+| emotion_score | Number | The confidence score for the emotion.                  | Will be computed only if emotion is computed.   |
+| intent        | String | Intent corresponding to the segment: Current intent list: `budget`, `authority`, `timing`, `action`, `need`. | Will be computed only if specified via metrics. |
```
Comment (sushant-hiray) [inline docs/interaction-analytics-api.md]: can you add `action`, `decision` to the list

File: docs/speaker-enrollment-api.md [hunk 2]
```diff
@@ -68,34 +68,34 @@ curl -X POST \
...
-| enroll_quality        | String | Quality of the enrollment               | values: poor, average, good, high  |
-| enrollment_complete   | Bool   | Status of the enrollment                | true if total speech exceeds 12sec |
-| total_speech_duration | Number | Total Speech Duration of the enrollment |                                    |
-| total_enroll_duration | Number | Total Duration of the enrollment        |                                    |
+| Parameter             | Type   | Description                              | Notes                                |
+| --------------------- | ------ | ---------------------------------------- | ------------------------------------ |
+| message               | String | Status of enrollment Success.            | Success:  enrollment is successfull. |
+| speaker_id            | String | Registered speaker id.                   |                                      |
+| enroll_quality        | String | Quality of the enrollment.               | values: poor, average, good, high.   |
+| enrollment_complete   | Bool   | Status of the enrollment.                | true if total speech exceeds 12sec.  |
```
Comment (sushant-hiray) [inline docs/speaker-enrollment-api.md]: `True`

File: docs/speaker-enrollment-api.md [hunk 3]
```diff
@@ -224,7 +224,7 @@ curl -X GET \
...
 ### Query Parameters
 
-| Parameter      | Type   | Description | Notes                                                                       |
-| -------------- | ------ | ----------- | --------------------------------------------------------------------------- |
-| apikey         | String | The apikey  | Required for authentication inside all requests                             |
-| getAllSpeakers | Bool   | true/false  | Set to `true` for fetching incomplete enrolled speakers too. Default to `false` |
+| Parameter      | Type   | Description | Notes                                                                            |
+| -------------- | ------ | ----------- | -------------------------------------------------------------------------------- |
+| apikey         | String | The apikey. | Required for authentication inside all requests.                                 |
+| getAllSpeakers | Bool   | true/false. | Set to `true` for fetching incomplete enrolled speakers too. Default to `false`. |
```
Comment (sushant-hiray) [inline docs/speaker-enrollment-api.md]: standardise true/false to the capitalized variant.

**Review by sushant-hiray — CHANGES_REQUESTED** [review]

File: docs/speaker-enrollment-api.md (hunk 3 above)
Comment (gargsanchit31) [reply docs/speaker-enrollment-api.md]: done

Comment (gargsanchit31) [reply docs/speaker-enrollment-api.md]: done

File: docs/speaker-enrollment-api.md (hunk 2 above)
Comment (gargsanchit31) [reply docs/speaker-enrollment-api.md]: done

File: docs/interaction-analytics-api.md (hunk 1 above)
Comment (gargsanchit31) [reply docs/interaction-analytics-api.md]: done

**Review by sushant-hiray — APPROVED** [review]

**Pull request merged**